tests/Database/QueryTest.php

```diff
@@ -93,6 +93,16 @@ public function setUp(): void
 			'password' => 'beatles',
 			'balance'  => 50
 		]);
+
+		$this->database->table('users')->insert([
+			'role_id'  => 4,
+			'username' => 'foo',
+			'fname'    => 'Foo',
+			'lname'    => 'Bar',
+			'email'    => 'foo@bar.com',
+			'password' => 'AND',
+			'balance'  => 0
```

Comment:
To ensure that the `min` test doesn't return `0` by accident:

```suggestion
			'balance'  => -30
```